sample-apps/react-native/dogfood/src/hooks/useAnonymousInitVideoClient.ts

```diff
@@ -1,39 +1,18 @@
 import { StreamVideoClient } from '@stream-io/video-react-native-sdk';
 import { useEffect, useState } from 'react';
 
-import { createToken } from '../modules/helpers/createToken';
 import { useAppGlobalStoreValue } from '../contexts/AppContext';
 
-type InitAnonymousVideoClientType = {
-  callId?: string;
-  callType?: string;
-};
+export const useAnonymousInitVideoClient = () => {
+  const apiKey = useAppGlobalStoreValue((store) => store.apiKey);
 
-export const useAnonymousInitVideoClient = ({
-  callId,
-  callType,
-}: InitAnonymousVideoClientType) => {
-  const appEnvironment = useAppGlobalStoreValue(
-    (store) => store.appEnvironment,
-  );
   const [client, setClient] = useState<StreamVideoClient>();
 
   useEffect(() => {
     let _client: StreamVideoClient | undefined;
     const run = async () => {
-      const anonymousUser = {
-        id: '!anon',
-      };
-      const { token, apiKey } = await createToken(
-        {
-          user_id: anonymousUser.id,
-          call_cids: `${callType}:${callId}`,
-        },
-        appEnvironment,
-      );
```

Comment:
cool! As we discussed.. anonymous works without token passing.. 

much simpler

Reply:
Thanks 😄